lib/Core/TableColumn.js

```diff
@@ -134,6 +149,11 @@ var TableColumn = function(name, values, options) {
 
 inherit(VariableConcept, TableColumn);
 
+function replaceValues(values, replaceWithZeroValues, replaceWithNullValues) {
+    // Replace "bad" values like "-" with zero, and "na" with null.
+    var replaced = values.map(function(value) { return (replaceWithZeroValues.indexOf(value) >= 0) ? 0 : value; });
+    return replaced.map(function(value) { return (replaceWithNullValues.indexOf(value) >= 0) ? null : value; });
```

Comment:
It might be nice to do this in a single pass over the array instead of 2, especially since each pass makes a copy of the array.